### PR TITLE
Merge the two branches

### PR DESCRIPTION
**Pull request by iuliaion**

### CODE REVIEW

File: u2f-gae-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# U2F Demo Deployment
+
+To deploy https://crxjs-dot-u2fdemo.appspot.com and configure the appspot
+server to the package Chrome app, you have to:
```
Comment (juanlang) [inline u2f-gae-demo/README.md]: "the package Chrome app": I'm not sure what you mean here. Just to keep the terminology straight, there are:
1. The cryptotoken component extension, i.e. the extension that's built into Chrome.
2. The U2F extension, which we use for development.
3. The gnubbyd packaged app, which is used internally by Google, and not relevant here.

Which do you mean?

File: u2f-gae-demo/war/WEB-INF/appengine-web.xml [hunk 2]
```diff
@@ -1,7 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <application>u2fdemo</application>
-  <version>crxjs</version>
+  
+  <!--
+    Version of the extension deployment.
+    Uncomment this if you want to deploy a server instance that uses
+    the package Chrome app and does not require installing the U2F Chrome extension.
```
Comment (juanlang) [inline u2f-gae-demo/war/WEB-INF/appengine-web.xml]: Same comment about "the package Chrome app": please clarify which you mean.

Comment (iuliaion) [inline u2f-gae-demo/war/WEB-INF/appengine-web.xml]: Done.

File: u2f-gae-demo/README.md (hunk 1 above)
Comment (iuliaion) [inline u2f-gae-demo/README.md]: Clarified. Thanks!

File: u2f-gae-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# U2F Demo Deployment
+
+To deploy https://crxjs-dot-u2fdemo.appspot.com and configure the appspot
+server to the U2F extension you have to:
+ (1) Set the version value to 'crxjs' in appengine-web.xml 
+    <version>crxjs</version>
+ (2) Set the extension id in u2f-api.js to 'kmendfapggjehodndflmmgagdbamhnfd'
+     u2f.EXTENSION_ID = 'kmendfapggjehodndflmmgagdbamhnfd';
+     
+To deploy https://noext-dot-u2fdemo.appspot.com and configure the appspot
```
Comment (juanlang) [inline u2f-gae-demo/README.md]: Nit: put the noext block first? It's also the default, so it should probably the first thing the reader sees.

File: u2f-gae-demo/war/WEB-INF/appengine-web.xml [hunk 4]
```diff
@@ -1,7 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <application>u2fdemo</application>
-  <version>crxjs</version>
+  
+  <!--
+    Version of the extension deployment.
+    Uncomment this if you want to deploy a server instance that uses the extension
+    that is built into Chrome.
```
Comment (juanlang) [inline u2f-gae-demo/war/WEB-INF/appengine-web.xml]: This seems to contradict what this is for. This is for the U2F extension, no?

Comment (iuliaion) [inline u2f-gae-demo/war/WEB-INF/appengine-web.xml]: Fixed! Thanks.

File: u2f-gae-demo/README.md (hunk 3 above)
Comment (iuliaion) [inline u2f-gae-demo/README.md]: Done.

File: u2f-gae-demo/war/WEB-INF/appengine-web.xml [hunk 5]
```diff
@@ -1,7 +1,20 @@
...
+  <!--
+    Version of the extension deployment.
+    Uncomment this if you want to deploy a server instance that uses the extension
+    that uses the U2F Chrome extension.
+
+    <version>crxjs</version>
+  -->
+
+  <!--
+    Version of the no-extension deployment.
```
Comment (juanlang) [inline u2f-gae-demo/war/WEB-INF/appengine-web.xml]: Should go first?

File: u2f-gae-demo/war/WEB-INF/appengine-web.xml [hunk 6]
```diff
@@ -1,7 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <application>u2fdemo</application>
-  <version>crxjs</version>
+  
+  <!--
+    Version of the extension deployment.
+    Uncomment this if you want to deploy a server instance that uses the extension
```
Comment (juanlang) [inline u2f-gae-demo/war/WEB-INF/appengine-web.xml]: "that uses the extension that uses the U2F Chrome extension": s/that uses the extension//

Comment (iuliaion) [inline u2f-gae-demo/war/WEB-INF/appengine-web.xml]: Fixed.

**Comment by juanlang** [comment]
lgtm, thanks very much Iulia! I'll note this partially addresses issue 49.

**Pull request merged**